File: app/src/main/java/com/blog/demo/third/UmengActivity.java
package com.blog.demo.third;

import android.app.Activity;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;

import com.blog.demo.R;
import com.umeng.analytics.MobclickAgent;

public class UmengActivity extends Activity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_third_umeng);

        findViewById(R.id.btn_click).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                MobclickAgent.onEvent(UmengActivity.this, "btn_click");
            }
        });
    }

    @Override
    protected void onResume() {
        super.onResume();
        MobclickAgent.onResume(this);
    }

    @Override
    protected void onPause() {
        super.onPause();
        MobclickAgent.onPause(this);
    }

}

File: app/src/main/java/com/blog/demo/third/UmengActivity.kt
package com.blog.demo.third

import android.app.Activity
import android.os.Bundle
import android.view.View
import com.blog.demo.R
import com.umeng.analytics.MobclickAgent

class UmengActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_third_umeng)
        findViewById<View>(R.id.btn_click).setOnClickListener {
            MobclickAgent.onEvent(this@UmengActivity, "btn_click")
        }
    }

    override fun onResume() {
        super.onResume()
        MobclickAgent.onResume(this)
    }

    override fun onPause() {
        super.onPause()
        MobclickAgent.onPause(this)
    }

}